packages/p2p/src/components/my-ads/quick-add-modal.jsx

```diff
@@ -73,9 +73,23 @@ const QuickAddModal = ({ advert }) => {
         }
     };
 
+    const hideModal = close_all => {
```

Comment:
Please start your booleans with `should_` or `is_` for clarity 

```suggestion
    const hideModal = should_close_all_modals => {
```